input/pagecontent/general-guidance.md

```diff
@@ -110,6 +110,295 @@ Example: Patient resource with interpreter required and language is known
 }
 ~~~
 
+### Representing body site and laterality
+When using some AU Core profiles it may be desirable to represent relevant body site and laterality information associated with a record using CodeableConcept elements, notably:
+*  [AU Core Condition](StructureDefinition-au-core-condition.html) - with primary finding code `Condition.code` and `Condition.bodySite`
+*  [AU Core Procedure](StructureDefinition-au-core-procedure.html) - with primary procedure code `Procedure.code` and `Procedure.bodySite`
+
+When recommending approaches to this, considerations include:
```

Comment:
The sentence "when recommending approaches to this, considerations include:" - I'm not sure who this is referring to, In other words is this a statement for the AU Core IG authors that we had to consider these things when we came up with the recommendations? or is it a set of considerations for implementations when handling body site or laterality information?

Please adjust this section to make clear which.

Reply:
@brettesler-ext consider tweaking further. I'm still not sure what it actually means.  Who is providing a recommended representation? Is that the implementation? Who is considering something?

Maybe?:

"Implementers should consider the following when implementing representations of body site / laterality:"

Reply:
I think, tweaking the opening of this section to make it shorter and clearer might be something like:

When exchanging Procedure and Condition resources using AU Core profiles it may be desirable to represent a relevant body site and associated laterality using CodeableConcept elements. In FHIR, body site and associated laterality can be recorded in various ways and implementers are encourage to consider the following points when implementing:
* The `bodySite` element is not *Must Support* in AU Core profiles, there is no expectation to fill or meaningfully consume this element.
* The `CodeableConcept.text` element is system populated and may reflect more specific detail than the `CodeableConcept.coding` concepts provided.

AU Core provides the following guidance for what to do in each of the following scenarios:
1. Primary finding/procedure code with body site and laterality as a pre-coordinated code.
2. Primary finding/procedure code with body site (without laterality) as a pre-coordinated code, and a separate laterality coded qualifier.
3. Coded body site with laterality and separate primary finding/procedure code.
4. Coded body site without laterality and separate coded laterality qualifier and a primary finding/procedure code.